joomla/joomla-cms
# [3.x] composer update --no-dev

### Summary of Changes

I have run composer update --no-dev
```
  - Updating joomla/string (1.4.3 => 1.4.4)
  - Updating joomla/filter (1.3.5 => 1.4.0)
  - Updating symfony/polyfill-php55 (v1.12.0 => v1.18.1)
  - Updating joomla/registry (1.6.2 => 1.6.3)
  - Updating symfony/polyfill-util (v1.12.0 => v1.18.1)
  - Updating symfony/polyfill-php56 (v1.12.0 => v1.18.1)
  - Updating symfony/polyfill-php71 (v1.12.0 => v1.18.1)
  - Updating symfony/polyfill-php73 (v1.12.0 => v1.18.1)
  - Updating google/recaptcha (1.1.2 => 1.2.4)
  - Updating brumann/polyfill-unserialize (v1.0.4 => v2.0.0)
  - Updating typo3/phar-stream-wrapper (v2.1.3 => v2.2.0)
  - Updating joomla/filesystem (1.5.2 => 1.6.0)
  - Updating symfony/polyfill-ctype (v1.12.0 => v1.18.1)
  - Updating symfony/yaml (v2.8.50 => v2.8.52)
  - Updating psr/log (1.1.1 => 1.1.3)

```

### Testing Instructions

- Make sure Google recaptcha still works
- Navigate the backend and frontend to make sure that the underlying packages still work
- install an extension to make sure the filysystem package does not break

### Actual result BEFORE applying this Pull Request

Outdated deps

### Expected result AFTER applying this Pull Request

up-to-date deps

### Documentation Changes Required

None.
